src/components/ContentGroup/ContentGroup.tsx

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import classNames from "classnames";
+
+import { CLASSPREFIX as eccgui } from "../../configuration/constants";
+import {
+    Divider,
+    Icon,
+    IconButton,
+    OverflowText,
+    Section,
+    SectionHeader,
+    Spacing,
+    StickyTarget,
+    StickyTargetProps,
+    Toolbar,
+    ToolbarSection,
+    Tooltip,
+} from "../index";
+
+export interface ContentGroupProps extends Omit<React.HTMLAttributes<HTMLElement>, "title"> {
+    title?: string;
+    level?: number;
+    contextInfo?: React.ReactElement | React.ReactElement[];
+    annotation?: React.ReactElement | React.ReactElement[];
+    actionOptions?: React.ReactElement | React.ReactElement[];
+    isCollapsed?: boolean;
+    handlerToggleCollapse?: () => void;
+    borderMainConnection?: boolean | string[];
+    borderSubConnection?: boolean | string[];
+    whitespaceSize?: "tiny" | "small" | "medium" | "large" | "xlarge";
+    dataTestId?: string;
+    minimumHeadlineLevel?: 1 | 2 | 3 | 4 | 5 | 6;
+    stickyHeaderProps?: Omit<StickyTargetProps, "children">;
+    contentClassName?: string;
+    description?: string;
+    hideGroupDivider?: boolean;
+}
+
+export const ContentGroup = ({
+    children,
+    className = "",
+    title,
+    contextInfo,
+    annotation,
+    actionOptions,
+    isCollapsed = false,
+    handlerToggleCollapse,
+    borderMainConnection = false,
+    borderSubConnection = false,
+    level = 1,
+    minimumHeadlineLevel = 3,
+    whitespaceSize = "small",
+    style,
+    dataTestId,
+    stickyHeaderProps,
+    contentClassName,
+    description,
+    hideGroupDivider,
+    ...otherHtmlProps
+}: ContentGroupProps) => {
+    const displayHeader = title || handlerToggleCollapse;
+
+    let borderGradient: string | undefined = undefined;
+    if (typeof borderSubConnection === "object") {
+        const borderColors: string[] = Array.isArray(borderSubConnection) ? borderSubConnection : [borderSubConnection];
+        borderGradient = borderColors.every((color) => CSS.supports("color", color))
+            ? borderColors
+                  .map((color, index) => {
+                      return (
+                          `${color} ` +
+                          `${(index / borderColors.length) * 100}% ` +
+                          `${((index + 1) / borderColors.length) * 100}%`
+                      );
+                  })
+                  .join(", ")
+            : undefined;
+    }
+
+    const contextInfoElements = Array.isArray(contextInfo) ? contextInfo : [contextInfo];
+
+    const headerContent = displayHeader ? (
+        <>
+            <SectionHeader className={`${eccgui}-contentgroup__header`}>
+                <Toolbar>
+                    {handlerToggleCollapse && (
+                        <ToolbarSection>
+                            <IconButton
+                                className={`${eccgui}-contentgroup__header__toggler`}
```

Comment:
We should add a tooltip, so we need translated terms here. The GUI elements do not use any specific translation library but you can integrate it by adding translation properties or callback functions. We have done both in the past, see:

* https://github.com/eccenca/gui-elements/blob/develop/src/components/TextField/SearchField.tsx
* https://github.com/eccenca/gui-elements/blob/develop/src/cmem/ActivityControl/SilkActivityControl.tsx